content/source/docs/enterprise/api/teams.html.md

```diff
@@ -10,6 +10,62 @@ sidebar_current: "docs-enterprise2-api-teams"
 
 The Teams API is used to create and destroy teams. The [Team Membership API](./team-members.html) is used to add or remove users from a team. To give a team access to a workspace use the [Team Access API](./team-access.html) to associate a team with privileges on a workspace.
 
+## List teams
+
+`GET organizations/:organization_name/teams`
+
+Parameter            | Description
+---------------------|------------
+`:organization_name` | The name of the organization to create the team in. The organization must already exist in the system, and the user must have permissions to create new teams.
```

Comment:
The description of the parameter doesn't seem accurate for the GET method, since it's not creating.

Reply:
copypasta gone wrong